docs/Using Fleet/segment-hosts.md

```diff
@@ -6,128 +6,52 @@
 ℹ️  In Fleet 4.0, Teams were introduced.
 ```
 
-- [View teams](#view-teams)
-- [Create a team](#create-a-team)
-- [Automatically adding hosts to a team](#automatically-adding-hosts-to-a-team)
+- [Overview](#overview)
+- [Naming conventions](#naming-conventions)
 - [Transfer hosts to a team](#transfer-hosts-to-a-team)
-- [Add users to a team](#add-users-to-a-team)
-- [Remove a member from a team](#remove-a-member-from-a-team)
-- [Remove a team](#remove-a-team)
+
+## Overview
 
 In Fleet, you can group hosts together in a team.
 
 With hosts segmented into exclusive teams, you can apply specific queries, policies, and agent options to each team.
 
-For example, you might create a team for each type of system in your organization. You can name the teams `Workstations`, `Workstations - sandbox`, `Servers`, and `Servers - sandbox`.
-
-> A popular pattern is to end a team’s name with “- sandbox”, then you can use this to test new queries and configuration with staging hosts or volunteers acting as canaries.
-
 Then you can:
 
 - Enroll hosts to one team using team specific enroll secrets
 
 - Apply unique agent options to each team
 
-- Schedule queries that target one or more teams
-
-- Run live queries against one or more teams
-
-- Grant users access to one or more
+- Schedule queries that target one or more team
 
-## View teams
+- Run live queries against one or more team
 
-To view teams:
+- Grant users access to one or more team
```

Comment:
I think this list might be repetitive. We already summarize what you can do w/ teams in this sentence: https://github.com/fleetdm/fleet/pull/17116/files#diff-4c458bc8bb8c8712cc4a491c9ac6f2a1a8c784fd1480cf67dfea806b50fe3178R17

Reply:
Agreed